src/G2/Config/Config.hs

```diff
@@ -93,6 +98,7 @@ mkConfig homedir = Config Regular
     <*> mkLogMode
     <*> flag Sharing NoSharing (long "no-sharing" <> help "disable sharing")
     <*> flag InstBefore InstAfter (long "inst-after" <> help "select to instantiate type variables after symbolic execution, rather than before")
+    <*> flag Lax Aggressive (long "show-types" <> help "determing whether you want to show more type ifnromation when you are logging the states")
```

Comment:
Typos: "determining", "information".  Since it's a boolean flag, maybe:
"set to show more type information when logging states"?